packages/migrations/src/utils/create-migration.ts

```diff
@@ -58,4 +59,62 @@ export function createMigration(
   }
 }
 
+export function createEdtrIoMigration({
+  exports,
+  migrateState,
+}: {
+  exports: any
+  migrateState: (state: any) => any
+}) {
+  createMigration(exports, {
+    up: async (db) => {
+      interface Row {
+        id: number
+        value: string
+        revisionId: number
+      }
+
+      const rows = await db.runSql<Row[]>(`
+        SELECT erf.id, erf.value, er.id as revisionId
+        FROM entity_revision_field erf
+          LEFT JOIN entity_revision er on erf.entity_revision_id = er.id
+          LEFT JOIN entity e on er.repository_id = e.id
+          LEFT JOIN type entity_type on entity_type.id = e.type_id
+        WHERE erf.field = 'content'
+          AND entity_type.name IN ("article", "course-page",
+                                   "event", "grouped-text-excercise",
+                                   "text-exercise", "text-exercise-group",
+                                   "text-solution")
```

Comment:
Why do we need only those types?

Reply:
The other types like `video` do not have edtr-io states, right?

Reply:
They do:
```sql
SELECT DISTINCT t.name
FROM entity_revision_field erf
         JOIN entity_revision er on erf.entity_revision_id = er.id
         JOIN entity e on er.repository_id = e.id
         JOIN type t on e.type_id = t.id
WHERE erf.value LIKE '%"plugin"%'
```